src/SFA.DAS.ApplyService.InternalApi/Startup.cs

```diff
@@ -107,6 +106,7 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<List<RoatpSequences>>(_configuration.GetSection("RoatpSequences"));
             services.Configure<List<CriminalComplianceGatewayConfig>>(_configuration.GetSection("CriminalComplianceGatewayConfig"));
             services.Configure<List<CriminalComplianceGatewayOverrideConfig>>(_configuration.GetSection("SoleTraderCriminalComplianceGatewayOverrides"));
+            services.Configure<List<CharityCommissionOuterApiAuthentication>>(_configuration.GetSection("CharityCommissionOuterApiAuthentication"));
```

Comment:
There should not be a list here

Reply:
Also we don't need to register this if the httpclient is correctly registered.

Reply:
Done